Content.Client/Chemistry/UI/TransferAmountWindow.xaml.cs

```diff
@@ -21,10 +20,10 @@ public TransferAmountWindow()
 
         public void SetBounds(int min, int max)
```

Comment:
im 99% sure its because casting fixedpoint2 to int isnt rounding down

Reply:
https://github.com/space-wizards/space-station-14/blob/cad1204a6da32fc5b75c1d7478fd8ab13f5f314a/Content.Client/Chemistry/UI/TransferAmountBoundUserInterface.cs#L29-L30

I solved the 100x value problem by just dividing the fixed point by 100.

![image](https://github.com/user-attachments/assets/5336442d-ffd5-478f-9fc1-cc1a3445f155)

Reply:
https://github.com/space-wizards/space-station-14/blob/master/Content.Shared/FixedPoint/FixedPoint2.cs#L196-L199

Oh I see, I shouldn't be using .Value

I should be using .Int()